helm/values.yaml.dist

```diff
@@ -1,7 +1,9 @@
 image:
-  repository: codelicia/turbo-enigma
+  repository: ghcr.io/codelicia/turbo-enigma
   pullPolicy: Always
-  tag: latest
+  tag: 0.1.0   # use a release tag (https://github.com/codelicia/turbo-enigma/tags)
```

Comment:
Does it make sense to use semantic versioning got the tag? or would using monotonic versioning (`r3`, `r4`, etc...) better?

Reply:
So far it's using semantic versioning for images based on tags manually created.

Would be nice to look at how it will evolve between the chart version (introduced in pull request #33) and the image version.